### ISSUE TITLE
Dota 2 crashes when loading into a map.

### ISSUE THREAD

**Issue by lucianofeder** [ValveSoftware/Dota-2#1852]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/lucianofeder/b19aea22b9988fbdb3dfbb27a6de2785/): 
* Have you checked for system updates?: [Yes]
* Are you using the latest stable video driver available for your system? [Yes]
* Have you verified the game files?: [Yes]

#### Please describe your issue in as much detail as possible:
Basically the whenever a try to join or watch a game it crashes. Please link any large pastes as a [Github Gist](https://gist.github.com/lucianofeder/b19aea22b9988fbdb3dfbb27a6de2785).

#### Steps for reproducing this issue:

1. Open Dota
2. Watch or join game
3. crash

**Comment by bielhxx** [comment]
The same is happening to me.
